### PR TITLE
Port copyright tool from Core repo.

### PR DESCRIPTION
**Pull request by sickpig**
This is a port of PR #8674 and #12075 with a small changes to adapt the tool to BU repo

### CODE REVIEW

**Comment by sickpig** [comment]
Travis build failure is due to PPA Ubuntu repositories being down. I will restart the failed jobs once Travis network problem will be fixed.

File: contrib/devtools/README.md [hunk 1]
```diff
@@ -7,22 +7,65 @@ clang-format.py
...
+
+This subcommand also handles copyright headers that have only a single year. In
+those cases:
+```
+// Copyright (c) <year> The Bitcoin Core developers
+```
+will be updated to:
+```
+// Copyright (c) <year>-<lastModifiedYear> The Bitcoin Core developers
+```
```
Comment (ptschip) [inline contrib/devtools/README.md]: maybe should change reference of Bitcoin Core developers to just "Bitcoin developers"

Comment (ptschip) [reply contrib/devtools/README.md]: nvm...i see you already did this in a later commit.

File: contrib/devtools/README.md [hunk 2]
```diff
@@ -7,22 +7,65 @@ clang-format.py
...
-would be changed to:
+copyright\_header.py
+====================
 
-```// Copyright (c) 2009-2015 The Bitcoin Unlimited developers```
-// Copyright (c) 2015-2017 The Bitcoin Unlimited developers
+Provides utilities for managing copyright headers of `The Bitcoin Core
+developers` in repository source files. It has three subcommands:
+>>>>>>> 159597a2b... [devtools] script support for managing source file copyright headers
+
```
Comment (ptschip) [inline contrib/devtools/README.md]: looks like you left a >>>>>>>>>> in

**Pull request merged**